### DISCUSSION TITLE
WIll new organization be coming?

### DISCUSSION THREAD

**Discussion by firoz3130** [community/community#52265] [GitHub Education]
I see not many organizations and octernship opportunities are available now. Will there be new organizations and projects now?Also, I wanted to know whether this Octernship Program has some timeline or not. 
I saw that different organizations and their own projects have different timelines. So isn't it possible for new organizations and projects to come?

**Comment by samus-aran** [comment]
Hi @firoz3130 👋 

We have new partners and projects enrolling soon so do stay tuned for more info!

**Reply by firoz3130** [reply]
Thanks for the reply.
Looking forward to it.
